.github/pull_request_template.md

```diff
@@ -3,7 +3,7 @@ Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-<**YOUR TICKET ID**>
 - [ ] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
 - [ ] Does this PR avoid breaking the API? 
     Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
-- [ ] Does the PR comply to REST HTTP result codes policy outlined in the [user guide](https://docs.xenit.eu/alfred-api/stable-user/rest-api/index.html#rest-http-result-codes)?
+- [ ] Does the PR comply to REST HTTP result codes policy outlined in the [user guide](https://docs.xenit.eu/alfred-api/user/rest-api/index.html#rest-http-result-codes)?
```

Comment:
Link does not resolve yet, perhaps until it is published

Reply:
Correct. It works now.